assets/wpuf/js/frontend-form.js

```diff
@@ -10,7 +10,19 @@
             });
         });
     };
-
+    $.fn.extend({
+        /**
+        * Custom jQuery serialize wrapper.
+        *
+        * When WordPress 5.6 increased the jQuery version to 3.5.1, the serialize function changed. Instead of
+        * sending spaces as "+", they are sent as "%20". This wrapper is for backwards compatibility.
+        *
+        * @since ###
+        */
+        weSerialize: function() {
+            return $( this ).serialize().replaceAll( '%20', '+' );
+        },
+    });

```

Comment:
Is there any way we can not duplicate this function? Is this the 3rd copy? 1 on the backend, 2 on the frontend?

Reply:
To test the broken state: make sure the master branch on your testing environment is up to date. 
Create or use a form with a Name and Text field.
And try to submit the form. The form will not submit and F12 will show the weSerialize is not a function error.

Reply:
@bwmarkle Updated the function to be on the frontend and back end just once based on what scripts are being enqueue'd

Reply:
To confirm, `weSerialize` is only defined in one place?

IE the `frontend-form.js` script is loaded in the admin as well?

Reply:
One will be on the frontend and one on the backend for this release.